src/config.py.template

```diff
@@ -15,8 +15,8 @@ USE_CLOUD_SERVICES = os.environ.get("USE_CLOUD_SERVICES", "False")
 
 # Scoring constants
 HPF_SCORING_REGION = os.environ.get("HPF_SCORING_REGION", "maven")
-MAX_COMPANION_REC_COUNT = int(os.environ.get("MAX_COMPANION_REC_COUNT", "5"))
-UNKNOWN_PACKAGES_THRESHOLD = float(os.environ.get("UNKNOWN_PACKAGES_THRESHOLD", "0.3"))
+# MAX_COMPANION_REC_COUNT = int(os.environ.get("MAX_COMPANION_REC_COUNT", "5"))
```

Comment:
Please remove these variables if you don't need them, if you do then why are these commented?